ios/RNSScreenStack.mm

```diff
@@ -434,6 +463,56 @@ - (void)setPushViewControllers:(NSArray<UIViewController *> *)controllers
   }
 
   UIViewController *top = controllers.lastObject;
+#ifdef RN_FABRIC_ENABLED
```

Comment:
Why is it all in `ifdef`? Aren't the addition of  `[screenController resetViewToScreen];` only changes between the implementations?